templates/arnold/render-linux-frames/job.template.json

```diff
@@ -62,8 +62,7 @@
                 },
                 "repeatTask": {
                     "displayName": "frame {fileNameWithoutExtension}",
-                    "commandLine": "sudo mkdir -m a=rwx \"$AZ_BATCH_TASK_WORKING_DIR/images\";/opt/solidangle/mtoa/2018/bin/kick -i \"$AZ_BATCH_TASK_WORKING_DIR/assets/{fileName}\" -dw -v 6 -of [parameters('outputFormat')] -o \"$AZ_BATCH_TASK_WORKING_DIR/images/{fileNameWithoutExtension}.[parameters('outputFormat')]\"",
-                    "userIdentity": {
+                    "commandLine": "/bin/bash -c 'sudo mkdir -m a=rwx \"$AZ_BATCH_TASK_WORKING_DIR/images\";mkdir \"$AZ_BATCH_TASK_WORKING_DIR/assets\";/opt/solidangle/mtoa/2018/bin/kick -i \"$AZ_BATCH_TASK_WORKING_DIR/assets/{fileName}\" -dw -v 6 -of [parameters('outputFormat')] -o \"$AZ_BATCH_TASK_WORKING_DIR/images/{fileNameWithoutExtension}.[parameters('outputFormat')]\"'",                    "userIdentity": {
                         "autoUser": {
                             "scope": "task",
```

Comment:
scope should probably be "pool" now as its more efficient than "task"

Reply:
updated to 'pool'